jsons/Policies.json

```diff
@@ -567,7 +573,7 @@
             {
                 "name": "Treasure Fleets",
                 "uniques": [
-                    "[+1 Gold, +1 Production, +1 Food] from every [Coast]",
+                    "[+1 Gold, +1 Production, +1 Food] from every [Coast] <in tiles without [Bonus resource]> <in tiles without [Luxury resource]> <in tiles without [Strategic resource]> <in tiles without [Improvement]>",
```

Comment:
`<in tiles without [resource]> <in tiles without [Improvement]>`

Reply:
Fixed it